Library/Formula/ansible.rb

```diff
@@ -156,13 +156,42 @@ class Ansible < Formula
   end
 
   #
-  # boto (for AWS support)
+  # boto/boto3 (for AWS support)
   #
   resource "boto" do
     url "https://pypi.python.org/packages/source/b/boto/boto-2.38.0.tar.gz"
     sha256 "d9083f91e21df850c813b38358dc83df16d7f253180a1344ecfedce24213ecf2"
   end
 
+  resource "boto3" do
```

Comment:
[ecs_cluster](http://docs.ansible.com/ansible/ecs_cluster_module.html)
[ecs_task](http://docs.ansible.com/ansible/ecs_task_module.html)
[ec2_taskdefinition](http://docs.ansible.com/ansible/ecs_taskdefinition_module.html)
[route53_facts](http://docs.ansible.com/ansible/route53_facts_module.html)

Not sure how to interpret 'should be enabled by default' - The module is available to use. if the user chooses to use it.